src/style.css

```diff
@@ -1 +1,83 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir-Book;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-family: Avenir-Heavy;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
```

Comment:
The @font-face rule is incorrectly structured. Each font-face declaration should be separate. You should not define multiple font-family and src properties within a single @font-face block.